composer.json

```diff
@@ -15,7 +15,8 @@
 	"require-dev": {
 		"woocommerce/woocommerce-sniffs": "0.1.0",
 		"suin/phpcs-psr4-sniff": "^2.2",
-		"bamarni/composer-bin-plugin": "^1.4"
+		"bamarni/composer-bin-plugin": "^1.4",
+		"symfony/console": "^3.4 | ^5.2"
```

Comment:
Is the ` | ^5.2` expected? It didn't install for me automattically when doing `composer install`, I end up running `composer require symfony/console`.

Reply:
Interesting. I actually ran `composer require symfony/console` and the file was updated automatically. I will double check.